pkgs/by-name/sn/snes9x/package.nix

```diff
@@ -116,13 +116,15 @@ stdenv.mkDerivation rec {
 
         Version build with ${interface} interface.
       '';
-
-      license = licenses.unfreeRedistributable // {
-        url = "https://github.com/snes9xgit/snes9x/blob/${version}/LICENSE";
+      license = lib.licenses.unfreeRedistributable // {
+        url = "https://github.com/snes9xgit/snes9x/blob/${finalAttrs.src.rev}/LICENSE";
       };
-      maintainers = with maintainers; [ qknight thiagokokada ];
```

Comment:
Did you remove `qknight` as maintainer by mistake or there is a reason?

Reply:
_D'oh_! It was a mistake.

I was looking at the [soft retirements issue](https://github.com/NixOS/nixpkgs/issues/290642) when editing other PRs.